packages/do-not-merge/test/test.ts

```diff
@@ -0,0 +1,172 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* eslint-disable node/no-extraneous-import */
+
+import myProbotApp from '../src/do-not-merge';
+import {resolve} from 'path';
+import {Probot, createProbot, ProbotOctokit} from 'probot';
+import {config} from '@probot/octokit-plugin-config';
+import nock from 'nock';
+import {describe, it, beforeEach} from 'mocha';
+import snapshot from 'snap-shot-it';
+
+const TestingOctokit = ProbotOctokit.plugin(config);
```

Comment:
What if we pulled this `TestingOctokit` into `gcf-utils/test-helpers` or something, wondering how we could avoid this boiler plate in all our tests.

Not a blocker on this PR.

Reply:
I don't mind a bit of copying. But, seems good to look into.